packages/types/src/jwtv2.ts

```diff
@@ -114,6 +114,12 @@ export interface JwtPayload extends CustomJwtSessionClaims {
    */
   sts?: SessionStatusClaim;
 
+  /**
+   * Feature claims this represent features, you can have multiple features in the same claim. e.g.: "impersonation,emails"
+   * @experimental
+   */
+  fea?: string | null;
```

Comment:
I think we will first need to have JWT versioning before proceeding with this addition

Reply:
I agree, I'll move this one to draft, and we can update it later with the versioning changes.